Plugin.php

```diff
@@ -64,6 +66,25 @@ public function register()
             $model->extendClassWith('RainLab\Translate\Behaviors\TranslatableModel');
         });
 
+        /*
+         * Add translation support to theme settings
+         */
+        ThemeData::extend(function ($model) {
+            $model->extendClassWith(TranslatableModel::class);
+
+            if (!$model->propertyExists('translatable')) {
+                $model->addDynamicProperty('translatable', []);
+            }
+
+            $model->bindEvent('model.afterFetch', function() use ($model) {
+                foreach ($model->getFormFields() as $id => $field) {
+                    if (!empty($field['translatable'])) {
```

Comment:
You assume the translatable property is true by default. This is a boolean, we should be able to set it to false and expect it NOT to be translatable.

Reply:
Most themes won't have this property in their configs, so by default we should expect that it does not even exist. But if it exists and it is true (what `!empty` guarantees) we add the corresponding field into translatable list.

Reply:
Yes, but down the line, if a field has the translatable property, someone would expect that changing its value to false would disable the translation for the field

Reply:
Yes, it is exactly how it works. 
If no `translatable` or `translatable=false` - no action, if `translatable=true` the field is picked up for translation.

Reply:
Oh, sorry, didn't realize `empty()` treated false value as empty.